### ISSUE TITLE
Create movie detail screen

### ISSUE THREAD

**Issue by Willjobs94** [Jherrera31/MovieStore#9]

**Comment by Jherrera31** [comment]
Oye @Willjobs94, nosotros vamos a utilizar base de datos para esto, o lo haremos como la vez pasada que siempre mostraba la misma informacion en la pantalla de detail

**Comment by Willjobs94** [comment]
No, ahora mismo vamos a enfocarnos en crear la estructura basica del proyecto utilizandolo como la otra vez, mas adelante le agregamos lo de la base de datos. 

Mientras tanto muestra la misma información en todas las pantallas de detail.

**Comment by Jherrera31** [comment]
ok
